pkg/frontend/admin_openshiftcluster_kubernetesobjects_get.go

```diff
@@ -24,12 +25,12 @@ func (f *frontend) getAdminKubernetesObjects(w http.ResponseWriter, r *http.Requ
 	log := ctx.Value(middleware.ContextKeyLog).(*logrus.Entry)
 	r.URL.Path = filepath.Dir(r.URL.Path)
 
-	b, err := f._getAdminKubernetesObjects(ctx, r)
+	b, err := f._getAdminKubernetesObjects(ctx, r, kubeactions.New(log, f.env))
```

Comment:
why pass this through when you can do the same in the function below? Not a biggie, just seems unnecessary.

Reply:
Yeah, I think you're right.

Reply:
pass log through to the child function